go.mod

```diff
@@ -5,7 +5,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/PuerkitoBio/purell v1.1.1
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
-	github.com/alecthomas/chroma v0.7.1
+	github.com/alecthomas/chroma v0.7.2-0.20200305040604-4f3623dce67a
```

Comment:
I don't want to add non-release versions unless I really have to.

Reply:
OK. Let's wait for releasing it.
Should I close this and recreate then, or leave and update it?

Reply:
Up to you.

Reply:
Ok. 
Although I don't have specific reasons, I will close this for now